spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/HqlQueryRenderer.java

```diff
@@ -341,9 +342,7 @@ public List<JpaQueryParsingToken> visitEntityWithJoins(HqlParser.EntityWithJoins
 		tokens.addAll(visit(ctx.fromRoot()));
 		SPACE(tokens);
 
-		ctx.joinSpecifier().forEach(joinSpecifierContext -> {
-			tokens.addAll(visit(joinSpecifierContext));
-		});
+		ctx.joinSpecifier().forEach(joinSpecifierContext -> tokens.addAll(visit(joinSpecifierContext)));
 
```

Comment:
I am usually a fan of shortening things like this, but since this is an ANTLR Visitor implementation, I actually prefer to retain it the way it is. It makes it easy when grammar tweaks require me to add another `tokens.addAll(...)` inside that clause. And since this pattern is used A LOT in this class (and its counterpart), it becomes easier to maintain if I retain the pattern consistently everywhere.